products/memcached.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Memcached
+category: db
+iconSlug: memcached
+permalink: /memcached
+versionCommand: memcached -h
+changelogTemplate: "https://github.com/memcached/memcached/wiki/ReleaseNotes{{'__LATEST__'|replace:'.',''}}"
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:generic/memcached
+-   purl: pkg:deb/ubuntu/memcached
+-   purl: pkg:deb/debian/memcached
+-   purl: pkg:rpm/amzn/memcached
+-   purl: pkg:rpm/redhat/memcached
+-   purl: pkg:rpm/centos/memcached
+-   purl: pkg:docker/library/memcached
+-   repology: memcached
+
+auto:
+-   git: https://github.com/memcached/memcached.git
+
+releases:
+-   releaseCycle: "1.6"
+    releaseDate: 2020-03-09
+    support: true
+    eol: false
+    latest: '1.6.22'
+    link: https://github.com/memcached/memcached/wiki/ReleaseNotes1622
+    latestReleaseDate: 2023-10-09
+
+-   releaseCycle: "1.5"
+    releaseDate: 2017-07-21
+    support: false
+    eol: true
```

Comment:
```suggestion
    eol: 2020-03-09
```

Given only the latest release is supported, N-1 release support date ends when the N release is released.